core/templates/components/question-directives/modal-templates/question-editor-save-modal.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal-header">
+  <h3>
+    <span>Save changes</span>
+  </h3>
+</div>
+<div class="modal-body oppia-long-text">
+  <p>
+    Please enter a brief description of what you have changed:
+    <textarea rows="3" cols="50" [attr.maxlength]="MAX_COMMIT_MESSAGE_LENGTH" [(ngModel)]="commitMessage" (focus)="saveChangesModalOpened" class="protractor-test-commit-message-input" autofocus></textarea>
```

Comment:
Just a note, please make sure to test that there are no change detection related issues with the enabling/disabling of the save button in this modal.

Reply:
I am applying the change detection manually wherever this component is used to avoid flake. ( See line 614 in questions-list.component.ts)

Reply:
Hm, in the code that you have pointed to, it seems like you are triggering the digest loop after the modal has closed. The confirm button on line 14 in this file seems to use commitMessage to enable/disable itself, can you please check if change detection works as expected (i.e. as the user types, the button enables itself and when the text is made empty, it is disabled).

Reply:
Will upload the video in a moment, (currently starting my server)

Reply:
The code that I pointed you is needed because without that question is not getting saved in one click(change detection issue). Uploaded the proof for working of save modal.